recipe/meta.yaml

```
@@ -1,36 +1,48 @@
-{% set version = "26.2.0" %}
+{% set version = "27.1.0" %}
+{% set name = "pyzmq" %}
 
 package:
-  name: pyzmq
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   # We use the pypi URL as it includes the prepared Cython files.
-  url: https://pypi.io/packages/source/p/pyzmq/pyzmq-{{ version }}.tar.gz
-  sha256: 070672c258581c8e4f640b5159297580a9974b026043bd4ab0470be9ed324f1f
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac0765e3d44455adb6ddbf4417dcce460fc40a05978c08efdf2948072f6db540
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: true  # [py<38]
 
 requirements:
   build:
-    - pkg-config 0.29.2  # [not win]
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake >=3.15
     - make  # [not win]
-    - ninja-base # [win]
+    - pkg-config 0.29.2  # [not win]
   host:
+    - pip
     - python
+    - cffi  # [python_impl == 'pypy']
+    - cython >=3.1.0  # [python_impl != 'pypy']
     - packaging
-    - pip
-    - scikit-build-core
-    - cython >=3.0.0
+    - scikit-build-core >=0.10
     - zeromq 4.3.5
   run:
     - python
-    - zeromq >=4.3.5,<4.4.0a0
+    - cffi  # [python_impl == 'pypy']
+    - zeromq # constraint through run_exports
+
+{% set ignore_tests = "" %}
+{% set tests_to_skip = "" %}
+# CI issue: zmq.error.ZMQError: Socket operation on non-socket
+# Works fine locally on the Docker instance.
+{% set ignore_tests = ignore_tests + " --ignore=tests/test_log.py" %}  # [linux64]
+# E AssertionError: Python process died with code 1
+# E where 1 = <ProcessForTeardownTest name='ProcessForTeardownTest-1' pid=74778 parent=74770 stopped exitcode=1>.exitcode
+{% set tests_to_skip = '-k "not test_process_teardown"' %}  # osx
```

I still want to believe it _should_ work on `osx`...

```suggestion
{% set tests_to_skip = '-k "not test_process_teardown"' %}  # [osx]
```

---

To confirm test_process_teardown failing on CI : https://package-build.anaconda.com/v1/graph/037f9ce0-ef63-48cb-a33e-c486aee57d2c

---

With tests_to_skip: https://package-build.anaconda.com/v1/graph/ad0532ae-1ef6-4804-85a9-575e3b4c1fcf